### PR TITLE
Bare Metal Forms and Helpers - RoR

### PR DESCRIPTION
**Pull request by Rhelli**
> This PR is for review by Microverse TSEs

# 🤟🏽 🎸 Bare Metal Forms and Helpers - RoR

This project follows [The Odin Project project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) on creating a simple site where you can create and edit users. The main focus of the project is on using Rails forms and form helpers.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -3,11 +3,24 @@ def new
...
+      render 'edit'
+    end
+  end
+
   def create
     @user = User.new(user_params)
 
     if @user.save
-      redirect_to users_url
+      redirect_to @user
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: redirect to new_user_path

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,7 +1,8 @@
-<!--
 <div class="title">
   <h1>Create A New User</h1>
 </div>
+
+<!--
 <div class="outer-container">
   <div class="form-container">
     <form action="/users" method="post" accept-charset="UTF-8">
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Here instead of `/users/`, please use `users_path`

**Comment by Rhelli** [comment]
Hi there,

Thanks for your review! I've taken a look at your suggestions and corrected the mistakes that were present, namely:

 - New user redirection now points towards the 'new_user_path' vs. @users that was in the old version
- The commented out 'form_tag' in the 'new.html.erb' view has now been changed from pointing to '/user/' to 'users_path'.
 
Please do let Qoosim or me know if there are any more changes due on this!

All the best,

Qoosim and Rory

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -3,11 +3,24 @@ def new
...
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to @user
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: You forgot to fix here too.
You should redirect to `edit_user_path` if user was saved

**Pull request closed without merging**